### ISSUE TITLE
Druid balance

### ISSUE THREAD

**Issue by mikonameton** [Valhalla-Gaming/Tracker#7177]
**Describe the issue you're having**:

**Explain how you expect it work**:

**Steps to reproduce the problem**:

1. Sunfire spell is deal 11100 overkill damage and not afected by anti magics and spells and armor and ..
balanc druid has most bugs and highest bug damages
2. this spell can kill other classes whit 2 hit and this is not normal this 2 hits its mean one spell 800k deall damage and no matter how is hero enemy  not affected by armor and another thing (huge huge bug )
3. 

**Links to Wowhead, YouTube, etc**:
http://www.wowhead.com/spell=93402/sunfire

**Comment by Aveta** [comment]
Do you have the trait Sunfire Burns and how many points in it? My Balance druid has no arti traits aside from the two basic ones and deals exactly as much dmg as the tooltip states.
Armor reduces only Physical dmg thus it can't reduce spell dmg : P

**Comment by mikonameton** [comment]
druids hase a most bugs in wow in this server and noe one care about it 
he killed me with 2 hit this is not good

**Comment by Aveta** [comment]
Also, overkill damage is something completely different (it's just how much damage it dealt over your hp, not a measure of OPness). 2shotting is one thing, it's most likely trait bug because my balance druid deals perfectly normal dmg. Instead of calling most bugged, how about you go into specifics? Lots of artifact traits are bugged into one side or another, same with spells. And don't say nobody cares about it - fixes TAKE TIME.

**Comment by mikonameton** [comment]
but 126889 crit damage can kill everything

**Comment by mikonameton** [comment]
if u want to defend  then tell to developers to let this bug ha?
i hate you people
 everyone wants to play with bug..

**Comment by Aveta** [comment]
No, I don't want to play with bugs, I reported quite a few myself, even positive ones, but I'm saying fixes take TIME. They can't just press a button and boom it's fixed.
If you hate us so much then just leave the server.

**Comment by mikonameton** [comment]
i hate u coz u want to play with bug and dont lie

**Comment by mikonameton** [comment]
if i want to stay then i stay  u are nothing

**Comment by Aveta** [comment]
https://github.com/Valhalla-Gaming/Tracker/issues/7151
And you tell me I want to play with positive bugs?
Scripting **takes time** and we don't have that many devs. I don't know if you English is bad but you seem like you don't understand what I'm saying or you are trying to twist my words to insult me.

**Comment by mikonameton** [comment]
but killing the enemy with 3 hit is a huge bug and you know that
yea my english is bad but 
i know this  everyone wants to say we know lot of things
 i want to see a fixed class 
even know hunter mark have same problem but every one say no that is good and no bug

killing a tank with 1600k by 3 hit  this is a huge huge bug

**Comment by mikonameton** [comment]
and i know what is Scripting 
but  devs must think about more importantn thing like druid battleground and artifacts 
i mean the most thing u must do in server in the day

**Comment by Aveta** [comment]
There are classes that are broken to the point of unplayablility and that's current focus of devs looking at changelog.
As I said, artifact traits can have a big impact there. My current full opener (which is more than 3 spells) with Incarnation popped can deal about 1,15 mil damage without crits. I have 17,3k spell power and no traits aside from the base.
